### DISCUSSION TITLE
No idea what this API 3 is! Help

### DISCUSSION THREAD

**Discussion by thbvintage** [etsy/open-api#954] [Q&A] [answered]
I recently had orders and I usually download the .csv file from etsy, then upload to my factory and all the orders will appear there. Now it says something about API v2 not working that I need to upgrade to API version 3, I am NOT a coder, just an ordinary seller, who now cannot upload any of my sales for my factory to make my orders. Is it etsy's fault? How do I upgrade to API 3, or must this be made by my factory to comply with the new coding or? I am so LOST! I called etsy yesterday, the girl had no clue what I am talking about and had no technical knowledge. What do I do now?

**Comment by Gareth064** [comment, marked as answer]
If you say you're not a coder then who built the thing that was downloading the CSV for you?

If it was your "factory" people the. I'd expect them to update it.

I would also have expected them to know that the old API was being replaced. It's been a work in progress for over 2 years now and "most" app developers knew it was happening.

On a side note, you can manually download your sales details from the Etsy website just fine..it might not be in the exact format that the factory needs but it's better than nothing.

**Reply by thbvintage** [reply]
Thank you so much! Yeah, on etsy you can easily download the CSV file. Since now it is API 3, you're right to say my factory needs to update their platform! So far, they have not gotten back to me since they are prob not open. I am going to take an old CVS file I got from my trash and key in manually the orders. It prob work that way. I just thought of that last night so we shall see, it will prob work for now until they comply with etsy API 3. Thank you!
